tensorflow/c/experimental/profiler/profiler.h

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef TENSORFLOW_C_EXPERIMENTAL_PROFILER_PROFILER_H_
+#define TENSORFLOW_C_EXPERIMENTAL_PROFILER_PROFILER_H_
+#include <stddef.h>
+#include <stdint.h>
+
+#include "tensorflow/c/c_api_macros.h"
+#include "tensorflow/c/tf_status.h"
+
+#define TP_MAJOR 0
+#define TP_MINOR 0
+#define TP_PATCH 1
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct TP_Profiler {
```

Comment:
Add documentation comments for each of the structs.

Reply:
done